### PR TITLE
Add support to unstructrued

### PR DESCRIPTION
**Pull request by thinkall**
## Why are these changes needed?

[Unstructured](https://github.com/Unstructured-IO/unstructured) provides open-source components for ingesting and pre-processing images and text documents, such as PDFs, HTML, Word docs, and [many more](https://unstructured-io.github.io/unstructured/bricks.html#partitioning).

Adding support to unstructrued will enable ingesting eml, docx, epub, pptx, xlsx, jpeg and png in RetrieveChat.

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [x] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by sonichi — COMMENTED** [review]
I took a quick look and didn't find structural problems.
Suggestions:
1. Enable tests in some environments. Right now the new tests are all skipped.
2. Document the features. I don't remember whether the previous pdf/other format support is documented. As you add more of these features, need to keep the doc up to date.
3. Could you find a group of reviewers interested in the RAG feature? I'm sure there are many of them. Start from the contributors and identify good reviewers among them.

**Review by pcdeadeasy — APPROVED** [review]
Changes look good  to me.

File: website/docs/Installation.md [hunk 1]
```diff
@@ -59,34 +59,51 @@ When running AutoGen out of a docker container, to use docker for code execution
...
 
 Example notebooks:
-[Optimize for Code Generation](https://github.com/microsoft/autogen/blob/main/notebook/oai_completion.ipynb),
+
+[Optimize for Code Generation](https://github.com/microsoft/autogen/blob/main/notebook/oai_completion.ipynb)
+
 [Optimize for Math](https://github.com/microsoft/autogen/blob/main/notebook/oai_chatgpt_gpt4.ipynb)
 
-* retrievechat
+- #### retrievechat
```
Comment (sonichi) [inline website/docs/Installation.md:75]: don't use - when using ####?

File: website/docs/Installation.md [hunk 2]
```diff
@@ -59,34 +59,51 @@ When running AutoGen out of a docker container, to use docker for code execution
...
+
+[Automated Code Generation and Question Answering with Retrieval Augmented Agents](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_RetrieveChat.ipynb)
+
 [Group Chat with Retrieval Augmented Generation (with 5 group member agents and 1 manager agent)](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat_RAG.ipynb)
 
-* mathchat
+[Automated Code Generation and Question Answering with Qdrant based Retrieval Augmented Agents](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_qdrant_RetrieveChat.ipynb)
+
+
+- #### mathchat
```
Comment (sonichi) [inline website/docs/Installation.md:100]: same.

Comment (thinkall) [reply website/docs/Installation.md:100]: Just wanted to make the section title more visible.

**Pull request merged**